app/backend/src/tests/test_bg_jobs.py

```diff
@@ -649,6 +652,88 @@ def test_process_send_onboarding_emails(db):
         )
 
 
+def test_process_send_reference_reminders(db):
+    # need to test:
+    # case 1: bidirectional (no emails)
+    # case 2: host left ref (surfer needs an email)
+    # case 3: surfer left ref (host needs an email)
+    # case 4: neither left ref (host & surfer need an email)
+
+    process_send_reference_reminders(empty_pb2.Empty())
+
+    # case 1: bidirectional (no emails)
+    user1, token1 = generate_user(email="user1@couchers.org.invalid", name="User 1")
+    user2, token2 = generate_user(email="user2@couchers.org.invalid", name="User 2")
+
+    # case 2: host left ref (surfer needs an email)
+    # host
+    user3, token3 = generate_user(email="user3@couchers.org.invalid", name="User 3")
+    # surfer
+    user4, token4 = generate_user(email="user4@couchers.org.invalid", name="User 4")
+
+    # case 3: surfer left ref (host needs an email)
+    # host
+    user5, token5 = generate_user(email="user5@couchers.org.invalid", name="User 5")
+    # surfer
+    user6, token6 = generate_user(email="user6@couchers.org.invalid", name="User 6")
+
+    # case 4: neither left ref (host & surfer need an email)
+    # host
+    user7, token7 = generate_user(email="user7@couchers.org.invalid", name="User 7")
+    # surfer
+    user8, token8 = generate_user(email="user8@couchers.org.invalid", name="User 8")
+
+    # case 5: neither left ref, but host blocked surfer, so neither should get an email
+    # host
+    user9, token9 = generate_user(email="user9@couchers.org.invalid", name="User 9")
+    # surfer
+    user10, token10 = generate_user(email="user10@couchers.org.invalid", name="User 10")
+
+    make_user_block(user9, user10)
+
+    with session_scope() as session:
+        # note host requests have one day longer age
```

Comment:
I don't understand this whole commit, the comment, the date changes, or why you removed all the check/asserts

Reply:
> the comment

`create_host_reference` creates a reference where the actual request is one day older than the input `reference_age`

> the date changes

Noticed a bug where the reference was put in the db as having been created before the host request was created. Oops.

> why you removed all the check/asserts

I was making sure there were 4 emails sent. But my `expected_emails` is of length 4, so it gets checked anyway.

Reply:
"Note, host requests are one day older than the reference"

Something like that?